__tests__/app.test.js

```diff
@@ -224,3 +226,25 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH:/api/articles/:article_id", () => {
+  test("202: Responds with the updated article with ammended votes property", () => {
+    const testPatch = {
+      inc_votes: 5,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(testPatch)
+      .expect(202)
```

Comment:
I would expect a 200 here - just essentially saying everything went well and below is the proof